src/command_modules/azure-cli-network/HISTORY.rst

```diff
@@ -6,6 +6,8 @@ Release History
 2.2.6
 +++++
 * Fix `network dns zone create`. Command succeeds even if the user has configured a default location. See #6052.
+* `network vnet peering create`: depecrated `--remote-vnet-id` for `--remote-vnet`. Passing a name instead of resource ID to either parameter does not crash the CLI.
```

Comment:
`network vnet peering create`: Deprecated `--remote-vnet-id`. Added `--remote-vnet` which accepts name or ID.

Reply:
I will update change. This is more concise, and I just noticed my typo.